docs/dev-guides/howto-release-helm-chart.md

```diff
@@ -93,9 +94,34 @@ helm repo index . --url https://azureiiot.blob.core.windows.net/helm/ --merge '<
 
 Documentation of `helm repo index` command can be found [here](https://helm.sh/docs/helm/helm_repo_index/).
 
-Now `azure-industrial-iot-0.4.0.tgz` file and updated `index.yaml` file can be uploaded to `helm` container of
+Now `azure-industrial-iot-0.4.0.tgz` and updated `index.yaml` files can be uploaded to `helm` container of
```

Comment:
why not azure-industrial-iot-0.4.1.tgz? isn't this the latest release?

Reply:
This is a dev guide so I didn't wanted to change all the numbers there. I can change them as well.

Reply:
Updated values for `2.8.2` release. Added note about version part in `tgz` file name.